estate/models/estate_property.py

```diff
@@ -0,0 +1,34 @@
+from odoo import fields, models
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "estate property"
+    name = fields.Char('Property Name', required=True)
+    description = fields.Text('Description', required=False)
+    postcode = fields.Char('Postcode', required=False)
+    date_availability = fields.Date('Available From', copy=False, default=fields.Datetime.add(fields.Datetime.now(), months=3))
+    expected_price = fields.Float('Expected Price', required=True)
+    selling_price = fields.Float('Selling Price', readonly=True, copy=False)
+    bedrooms = fields.Integer('Bedrooms', default=2)
+    living_area = fields.Integer('Living Area (sqm)', default=1)
+    facades = fields.Integer('Facades', default=0)
+    garage = fields.Boolean('garage', default=False)
+    garden = fields.Boolean('garden', default=False)
+    garden_area = fields.Integer('garden Area (sqm)', default=0)
+    garden_orientation = fields.Selection(
+        string='Garden Orientation',
+        selection=[('north', 'North'), ('south', 'South'), ('east', 'East'), ('west', 'West')],
```

Comment:
Odoo guideline (not sure it's an official one though): prefer simple quote for "code" strings, and double quotes for displayed ones
```suggestion
        selection=[('north', "North"), ('south', "South"), ('east', "East"), ('west', "West")],
```